ci/docker/release-tool.Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM microsoft/dotnet:2.2-sdk as build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-bionic as build
```

Comment:
Why is this specifically using the bionic image?

Reply:
Because the regular one doesn't work 😢; the Git library fails to find one of the binaries and somehow it only works on this image.

Reply:
What specifically broke? And how did you find this image as the solution?

Reply:
https://github.com/libgit2/libgit2sharp/issues/1703
I tried using alpine, but it didn't work.

Reply:
This bionic image would only be required in the step that runs in then? This step likely wont need it.